silk/hw/hw_module.py

```diff
@@ -48,15 +48,18 @@ class HwModule(object):
     Maintains usage of a hardware resource
     """
 
-    def __init__(self, name, parser, node_id, model=None, interface_serial=None,
-                 interface_number=None, port=None, dut_serial=None, jlink_serial=None):
+    def __init__(self, name, parser, node_id, layout_center=None, layout_radius=None,
+                 model=None, interface_serial=None, interface_number=None, port=None,
+                 dut_serial=None, jlink_serial=None):
         self._name = name
         self._parser = parser
         self._claimed = False
         self._port = port
         self._model = model
         self._dut_serial = dut_serial
         self._node_id = node_id
+        self._layout_center = layout_center
+        self._layout_radius = layout_radius
```

Comment:
make sure they are both specified or both not? 
```suggestion
        assert (layout_center is None) == (layout_radius is None)
        self._layout_center = layout_center
        self._layout_radius = layout_radius
```